src/cascadia/TerminalSettingsEditor/CommonResources.xaml

```diff
@@ -29,7 +29,7 @@
     <Style x:Key="SettingsStackStyle"
            TargetType="StackPanel">
         <Setter Property="HorizontalAlignment" Value="Left" />
-        <Setter Property="Margin" Value="13,0,0,48" />
+        <Setter Property="Margin" Value="13,0,13,48" />

```

Comment:
This seems like a global change to all settings pages?

Reply:
Ah. This was leftover from an earlier implementation. Good catch.